eng/pipelines/live-test-cleanup.yml

```diff
@@ -53,12 +53,46 @@ stages:
           -Verbose
         displayName: Clean up subscription resources (AzureChinaCloud)
 
+      - template: /eng/common/TestResources/build-test-resource-config.yml
+        parameters:
+          SubscriptionConfigurations:
+            - $(sub-config-azure-cloud-test-resources)
+            - $(sub-config-communication-services-cloud-test-resources)
       - pwsh: |
           $subscriptionConfiguration = @'
-            $(acs-subscription-configuration)
+            $(SubscriptionConfiguration)
           '@ | ConvertFrom-Json -AsHashtable
 
           ./tools/live-test-cleanup/cleanup.ps1 `
           @subscriptionConfiguration `
           -Verbose
-        displayName: Clean up subscription resources (AzureCloud ACS Subscription)
\ No newline at end of file
+        displayName: Clean up subscription resources (AzureCloud ACS Subscription)
+
+      - template: /eng/common/TestResources/build-test-resource-config.yml
```

Comment:
Why are we using the join for a single sub?

Reply:
Because the configuration for this sub exists across two configurations. The acs-team controlled config contains their sub ID, and the engsys controlled config contains the azure-sdk-tests service principal credentials.

Reply:
I only see one sub listed in the list below.

Reply:
...I'm not even sure how this change made it into this PR. This should be a separate PR. Guess I had too many branches in flight last week.

Reply:
Fixed and moved to https://github.com/Azure/azure-sdk-tools/pull/1589